### PR TITLE
Added serial consistency for writer operations

### LINKED ISSUE #238: Setting Serial consistency for write operation
All the write operation are run as LWT. This needs Serial consistency to be set.

By default set serial consistency to SERIAL. In case the write consistency is set as LOCAL_QUORUM set the serial consistency to LOCAL_SERIAL.

### PR DESCRIPTION
**Pull request by maheshrajamani**
**What this PR does**:
Set serial consistency for write operations
**Which issue(s) this PR fixes**:
Fixes #238 #344 

**Checklist**
- [x] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Review by tatu-at-datastax — APPROVED** [review]

**Review by tatu-at-datastax — APPROVED** [review]
LGTM

File: src/test/java/io/stargate/sgv2/jsonapi/testresource/SerialConsistencyOverrideProfile.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package io.stargate.sgv2.jsonapi.testresource;
+
+import com.google.common.collect.ImmutableMap;
+import io.quarkus.test.junit.QuarkusTestProfile;
+import java.util.Map;
+
+public class SerialConsistencyOverrideProfile implements QuarkusTestProfile {
```
Comment (ivansenic) [inline src/test/java/io/stargate/sgv2/jsonapi/testresource/SerialConsistencyOverrideProfile.java:7]: nit: just move this as inner class of the test `SerialConsistencyOverrideOperationTest.java`, it's way easier to see immediately what the profile is.. That's how we did it so far, very easy to understand if you see the profile at the top of the test class immediatelly..

**Review by ivansenic — APPROVED** [review]
Perfect...

File: src/test/java/io/stargate/sgv2/jsonapi/testresource/SerialConsistencyOverrideProfile.java (hunk 1 above)
Comment (maheshrajamani) [reply src/test/java/io/stargate/sgv2/jsonapi/testresource/SerialConsistencyOverrideProfile.java]: Make sense, done.

**Pull request merged**